### PR TITLE
[##97631496] Add SSP create-draft features for all lots

### PR DESCRIPTION
**Pull request by TheDoubleK**
See: https://www.pivotaltracker.com/story/show/97631496

This adds features for all lots that create a new draft, complete all the pages (with a "happy path" - should be no errors), navigate to the summary page and delete the service to clean up.

The IaaS feature additionally checks that the correct values are displayed on the summary page - this is the same as we had for the G6 tests.  We might want to add summary page checking to the other lots too, but it would probably be overkill.

I have also copied the validations feature from the G6 tests, but have not had time to fix it to work with the new page and field names, hence the whole feature is marked as WIP.  I have only included the `ssp_steps` needed for the happy-path journeys here, so implementing the validation checking will probably require extra steps that can be found in the deleted methods here: https://github.com/alphagov/digitalmarketplace-smokey/commit/d12c2153d3f76507d1ed8a76487de7d785f4b6d1)

### CODE REVIEW

File: features/ssp_iaas_service.feature [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+
+  Background: Log in as a supplier
+    Given I am on the 'Supplier' login page
+    When I login as a 'Supplier' user
+    Then I should be on the supplier home page
+
+  Scenario: Select lot
+    Given I am at '/suppliers'
+    When I click 'Apply to become a G-Cloud 7 supplier and add services'
+    And I click 'Add a service'
```
Comment (quis) [inline features/ssp_iaas_service.feature]: This is going to fail because the 'Add service' link is no longer on this page.

Comment (TheDoubleK) [inline features/ssp_iaas_service.feature]: Well it was there yesterday afternoon!

Comment (quis) [inline features/ssp_iaas_service.feature]: Haha, sorry Kev. Consider it a note to self to fix this.

Comment (quis) [inline features/ssp_iaas_service.feature]: Should be addressed by 409fdee

**Pull request merged**